pkg/landscaper/execution/reconcile.go

```diff
@@ -32,12 +34,14 @@ type executionItem struct {
 
 // Reconcile contains the reconcile logic for a execution item that schedules multiple DeployItems.
 func (o *Operation) Reconcile(ctx context.Context) lserrors.LsError {
+	logger, ctx := utils.FromContextOrNew(ctx)
+	logger = logger.WithValues("operation", "reconcile", "resource", kutil.ObjectKeyFromObject(o.exec).String())
```

Comment:
we should use the constants for the keys from the logging package.

Reply:
There is actually a helper function for logging the beginning of a reconcile. Depending on whether you already have a logger or not, it is called as `logging.StartReconcileFromContext` or `logger.StartReconcile`. It takes a `reconcile.Request` as argument, which isn't present here, but maybe we can fake it?

The mentioned helper function doesn't log the operation, you can use `WithValues` for that. The `constants` package should already contain a key for `operation`.

Reply:
I changed this and log the method name using the `KeyMethod` constant.